pkg/apis/planetscale/v2/vitesscluster_defaults.go

```diff
@@ -108,3 +109,41 @@ func defaultClusterBackup(backup *ClusterBackupSpec) {
 		backup.Engine = defaultBackupEngine
 	}
 }
+
+func defaultTopoReconcileConfig(vt *VitessCluster) {
+	// Defaulting registration code.
+	if vt.Spec.TopoReconciliation.RegisterCells == nil {
+		registerCells := true
+		vt.Spec.TopoReconciliation.RegisterCells = &registerCells
```

Comment:
Use the utility function `pointer.BoolPtr(true)`.

Reply:
Done https://github.com/planetscale/vitess-operator/pull/26/commits/b026aec4a4017c54b1651342e80dc811ff55c00c6.